src/cdk/v2/destinations/pinterest_tag/procWorkflow.yaml

```diff
@@ -41,6 +43,13 @@ steps:
     condition: $.outputs.messageType === {{$.EventType.TRACK}}
     template: |
       $.assert(.message.event || .message.name, "event_name could not be mapped. Aborting.")
+
+  - name: isAdvertisementMappingEnabled
```

Comment:
Is it `advertising` or `advertisement`? We seem to use the terms interchangeably.

Let's be consistent in all the places.